ReleaseHistory.md

```diff
@@ -14,56 +14,49 @@
 - UEE => eliminate unhandled exceptions in engine
 
 ## **v4.1.0** UNRELEASED
+* BUG: Fix BA2004 HASH not SHA256 message - was hardcoded to SHA-1. [#865](https://github.com/microsoft/binskim/pull/865)
```

Comment:
>BUG [](http://example.com/codeflow?start=2&length=3)

Oops, this bug and all other bugs need to actually be grouped together, see note below